src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -378,8 +378,8 @@ protected function mergeWithExisting($entitiesByModule)
             // Merge
             if ($existingMessages) {
                 $entitiesByModule[$module] = array_merge(
-                    $existingMessages,
-                    $messages
+                    $messages,
```

Comment:
Oh, this is all about merging keys, I don't care about keys, I care about values. Seems we need a different mode for what I want.